src/value/parse.rs

```diff
@@ -15,10 +15,7 @@ pub fn is_whitespace(&byte: &char) -> bool {
 
 #[inline(always)]
 fn is_not_separator(&byte: &char) -> bool {
-    match byte {
-        ',' | '{' | '}' | '[' | ']' => false,
-        _ => true
-    }
+    !matches!(byte, ',' | '{' | '}' | '[' | ']')
```

Comment:
I think this looks ugly, especially the `!matches!`. I'm open to changing it back or to something else.

Reply:
I'm okay with it.